tests/figE.ipynb

```diff
@@ -0,0 +1,296 @@
+      "    Package may be exclusively used for academic and research purposes.\n",
+      "    No other types of uses are allowed;\n",
+      "    4. Any use of visualizations, charts, graphs, graphics, pictographs,\n",
+      "    plots, or other displays created via the R Package in an academic or\n",
+      "    research publication, including in a paper, manuscript, preprint, website,\n",
+      "    web service, or any other media material must be in conformity with the\n",
+      "    GISAID Publishing Guidelines, available at https://www.gisaid.org/publish,\n",
+      "    and the DAA, available at https://www.gisaid.org/daa; and\n",
+      "    5. By using the R Package you reaffirm your understanding of these\n",
+      "    terms and the DAA.\n",
+      "    When using this data, please state, \"This data was obtained from GISAID via the outbreak.info API\". \n",
+      "    WE DO NOT SUPPORT THIRD PARTY APPLICATIONS. THIS PACKAGE IS MEANT FOR RESEARCH AND VISUALIZATION PURPOSES ONLY. \n",
+      "    If you want to build third party applications, please contact GISAID via https://www.gisaid.org/help/contact/.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from outbreak_data import authenticate_user\n",
+    "authenticate_user.authenticate_new_user()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25f9074f-fe8a-4dfe-851f-2c234ca2cb0e",
+   "metadata": {},
+   "source": [
+    "First we specifiy the location and time of data collection. We're going to use wastewater samples taken in the state of Ohio collected over a 3 month period. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "59a9362a-9e54-4ef2-87f6-c354608e0826",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "state = \"Ohio\"\n",
+    "state_id = 'USA_US-OH'\n",
+    "startdate, enddate = \"2023-07-02\", \"2023-10-02\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "957b7475-e716-4892-9e6f-06ccf9f3c32e",
+   "metadata": {},
+   "source": [
+    "These functions are highly recommended to ensure smooth plotting of the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77f7fe03-b9f6-4eb4-bcd7-3720c1af4d75",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def binom_kernel(n, squeeze=0):\n",
+    "    return np.array([scipy.special.binom(n+2*squeeze, x) for x in np.arange(n)+squeeze])\n",
+    "\n",
+    "def lognorm_ci(x, r):\n",
+    "    return (np.exp(np.log(x)-(r**0.5/x)), np.exp(np.log(x)+(r**0.5/x)))\n",
+    "\n",
+    "smooth = binom_kernel(7*5, 35)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e2ed603-3042-49ca-865e-a823287bdeb8",
+   "metadata": {},
+   "source": [
+    "Now we go ahead and query for wastewater data using our defined specifications. After this, we'll need to organize our retrieved sample data by date and site_id within our specified region to get the viral load smaple data."
```

Comment:
Small typo here. Also good to explain that we're filtering out viral load information from sites with few samples, and then normalizing each site's viral load signals to have a variance of 1.